packages/common/src/objects/set-prop.ts

```diff
@@ -0,0 +1,15 @@
+import { deepSet } from "./deep-set";
+
+/**
+ * Sets a deep object property, constructing the property path as necessary
+ *
+ * @param path - the path to the property we want to set
+ * @param val - the value we want to set it to
+ * @param obj - the target object
+ */
+export function setProp(path: string | string[], val: any, obj: any) {
```

Comment:
what's the diff w/ this and [deepSet()](https://esri.github.io/hub.js/api/common/deepSet/)?

Reply:
nvm, I see you've updated it to use `deepSet()` now.